server/container/guice/protocols/jmap/src/test/java/org/apache/james/modules/TestJMAPServerModule.java

```diff
@@ -33,6 +33,8 @@
 import com.google.inject.name.Names;
 
 public class TestJMAPServerModule extends AbstractModule {
+    public static final int MAXIMUM_MESSAGE_COUNT = 10;
+    public static final TestJMAPServerModule DEFAULT = new TestJMAPServerModule(MAXIMUM_MESSAGE_COUNT);
```

Comment:
why do we need this 10 messages limitation everywhere ?

Reply:
It is used to provide a default value when user do not provide limits for GET MESSAGE LIST.

In prod a constant bound to 256 is used.

So no, we do not even need it...